packages/rrweb/src/record/index.ts

```diff
@@ -54,17 +54,13 @@ import {
 } from './error-handler';
 export type { CanvasManagerConstructorOptions } from './observers/canvas/canvas-manager';
 
-function wrapEvent(e: event): eventWithTime {
-  const eWithTime = e as eventWithTime;
-  eWithTime.timestamp = nowTimestamp();
-  return eWithTime;
-}
-
```

Comment:
Added in #124, but removed in [upstream PR](https://github.com/rrweb-io/rrweb/pull/1441/files#diff-ad746ef2bb1aa2d230a3b8635d3fd20a4587dc176bb71f42065a857392b755b3L43-L48) which builds upon #124.